src/loader.rs

```diff
@@ -391,13 +405,12 @@ impl<'b, 'w> Loader<'b, 'w> {
         &mut self,
         video_clip_obj: object::video::ClipHandle<'_>,
     ) -> anyhow::Result<Image> {
-        trace!("Loading texture image: {:?}", video_clip_obj.name());
+        info!("Loading texture image: {:?}", video_clip_obj.name());
```

Comment:
```suggestion
        debug!("Loading texture image: {:?}", video_clip_obj.name());
```

This is too verbose, info log level is visible by default. This only serves to check if the function ran, and the message is printed a lot. At most, I'd put it to `debug` level.